### PR TITLE
added includes

### PR DESCRIPTION
**Pull request by themasch**
Compiling on linux with gcc 4.7.3 fails for me without these imports. I guess string.h might be a linux-only thing so I wrapped it to prevent errors on windows. stdexcept on the other hadn should be everywhere. 

This needs testing on other OS before getting pulled, thanks!

### CODE REVIEW

File: src/libOL/Blocks/Block.h [hunk 1]
```diff
@@ -9,6 +9,10 @@
 #include <iostream>
 #include <vector>
 
+#ifdef __linux 
+#include <string.h>
```
Comment (toulouse) [inline src/libOL/Blocks/Block.h]: also valid for OS X

**Comment by themasch** [comment]
moved stdexcept to Blowfish.cpp as discussed in IRC. sry for these two useless commits on master. blame me.

**Comment by lukegb** [comment]
Erp, did you include `string.h` for `size_t`?

As a rule, you should prefer `cstring` over `string.h`, since the former is a C++ header.

**Comment by themasch** [comment]
you are late luke: https://github.com/loldevs/libOL/blob/master/src/libOL/Blocks/Block.h#L11  ;)

**Comment by lukegb** [comment]
Whoops, was reading the old commit diff not the new one.

**Pull request merged**